src/components-examples/journey-maps/angular/journey-maps-map-navigation/journey-maps-map-navigation-example.html

```diff
@@ -0,0 +1,63 @@
+<sbb-notification type="info" class="api-key-info">
+  This example uses a test API key, that might be disabled at any time.
+</sbb-notification>
+<div class="web-map">
+  <sbb-journey-maps
+    [apiKey]="apiKey"
+    #advancedMap
+    language="en"
+    (mapCenterChange)="mapCenterChange.next($event)"
+    (mapBoundsChange)="mapBoundingBoxChange.next($event)"
+    (zoomLevelsChange)="zoomLevels = $event"
+  ></sbb-journey-maps>
+</div>
+<div class="sbb-navigation">
+  <div class="row-1">
```

Comment:
On narrow screens, the bounding box and zoom info is cut off. 
Can you force line breaks here?